04.ls/ls.rb

```diff
@@ -9,6 +9,14 @@
 
 FILE_TYPE_LIST = { '01' => 'p', '02' => 'c', '04' => 'd', '06' => 'b', '10' => '-', '12' => 'l', '14' => 's' }.freeze
 
+# directory_pathsには複数のpathを指定することは許容しているが、現時点でファイル名を表示するのは1番目に指定したディレクトリのみにしている。
+def main
+  options, directory_paths = parse_options(ARGV)
+  target_directory_path = directory_paths[0] || './'
```

Comment:
```suggestion
  # directory_pathsには複数のpathを指定することは許容しているが、現時点でファイル名を表示するのは1番目に指定したディレクトリのみにしている。
  directory_path = directory_paths[0] || './'
```

コメントを書くなら対象のコードに一番近いところがいいと思います。

あと、「ターゲットじゃないdirectory_path」があるわけではないので、`target_`は削って良さそうです

Reply:
コメントの位置を変更しました。

確かにおっしゃる通り、ターゲットじゃないものはないのでかえってややこしい表現ですね。
削りました！